notes-apps-ogechukwu-okoli.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
```

Comment:
For best practices, you should have spaces between classes, functions, and some other blocks of code. Please give them some space